cpp/velox/substrait/SubstraitToVeloxPlanValidator.cc

```diff
@@ -432,6 +432,10 @@ bool SubstraitToVeloxPlanValidator::validate(const ::substrait::FetchRel& fetchR
   if (fetchRel.has_input()) {
     topNFlag = fetchRel.input().has_sort();
     if (topNFlag) {
+      if (fetchRel.offset() != 0) {
```

Comment:
@ulysses-you Do we need this check in native side? It seems we have validate it in scala side [here](https://github.com/oap-project/gluten/pull/4607/files#diff-5024a9ba02c16355af248920e6bd13de93a1149d1e0510e10bc6f7d31de9045fR50).

Reply:
I think we can remove it